WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -106,7 +106,6 @@ private async IAsyncEnumerable<Alice[]> CheckTxoSpendStatusAsync(Round round, [E
 					.Select(x => (Alice: x, StatusTask: Rpc.GetTxOutAsync(x.Coin.Outpoint.Hash, (int)x.Coin.Outpoint.N, includeMempool: true, cancellationToken)))
 					.ToList();
 
-				cancellationToken.ThrowIfCancellationRequested();
```

Comment:
After upgrading NBitcoin, this feels redundant.

Reply:
Yes. Feel free to remove it.